src/Microsoft.ML.Core/Data/Repository.cs

```diff
@@ -340,7 +340,9 @@ public Entry CreateEntry(string dir, string name)
 
             Stream stream;
             if (pathTemp != null)
+            {
```

Comment:
This is probably counter to other coding guidelines. But it seems that in this repository we are not using braces for single line blocks. Can you please revert this to stay aligned with that?

Reply:
Reverted the braces